hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/util/Clock.java

```diff
@@ -23,7 +23,11 @@
 
 /**
  * A simple clock interface that gives you time.
+ * @deprecated This class has been moved to {@link org.apache.hadoop.util.Clock}
+ * in hadoop-common-project and will be removed in Hadoop 4.0.0.
```

Comment:
Let's not mention 4.0.0 (or any other specific version). There isn't a specific release plan for a 4.0.0 yet. It's enough to say that the intent is to remove it someday.

Reply:
Thanks for the suggestion! That makes sence. I'll update the comments to remove the version reference and just indicate the intent to remove it in the future.